lib/orbit/user.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Orbit.User do
```

Comment:
**question (high):** Is this branch rebased against `main`? `main` has a `Orbit.Authentication.User`.

Reply:
It is based off of the latest `main`, I just didn't realize that module already existed. 🤦 I'll remove my duplicate implementation.

Reply:
Okay fixed in ab2dfbd (see also f776841 for another tiny improvement).